team/templates/index.html

```diff
@@ -1,21 +1,16 @@
-
-
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>React Redux Starter Kit</title>
-  <style type="text/css">
-  </style>
-</head>
-<body>
-  <div id="app"></div>
-
-  <!-- Attach React components -->
-  <script type="text/javascript" src="{{ url_for('static', filename='bundle.js') }}"></script>
```

Comment:
This is gonna break the React-webpack integration and give the front end people a lot of problems

Reply:
I see what you did here and I understand why.  For now, it may be best to move the stuff that was to another file (e.g. react_index) and deal with later.  Talk to one of the front-end people though (I suggest @Meggolbek)

Reply:
I am having issues without this file. I am trying to find a workaround.